externals/kyuubi-hive-sql-engine/src/test/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationSuite.scala

```diff
@@ -30,4 +31,14 @@ class HiveOperationSuite extends HiveEngineTests {
   override protected def jdbcUrl: String = {
     "jdbc:hive2://" + HiveSQLEngine.currentEngine.get.frontendServices.head.connectionUrl + "/;"
   }
+
+  test("test get query id") {
```

Comment:
Is it ok to write test case for getQueryId here?
I choose to do it because statement should be instanced to KyuubiStatement to cover this function.